src/components/commitment/Modals/CommitmentModal.js

```diff
@@ -126,6 +123,19 @@ const CommitmentModal = (props) => {
               label="immediately"
             />
           </DataGridCell>
+          {showCalendar && (
+            <>
+              <DataGridCell className={label}>Notificaton</DataGridCell>
+              <DataGridCell>
+                <Checkbox
+                  label="send mail on confirm"
```

Comment:
```suggestion
                  label="send mail on delayed confirmation"
```

IIRC, the confirmation mail can only happen, when `confirm_by` is set - so a delayed confirmation is set. If this is correct, it should be clear to the customer that he won't get mails for commitments which should become active immediately?

Reply:
Got it, thanks for explaining :-)